src/bgw/scheduler.c

```diff
@@ -488,7 +488,11 @@ ts_update_scheduled_jobs_list(List *cur_jobs_list, MemoryContext mctx)
 	{
 		ListCell *ptr;
 
+#if PG13_GE
```

Comment:
I suggest we create a compat version of `for_each_cell` too in order to make this code look nicer.